openedx/course-discovery
# feat: add management command to add course tags from csv

# [PROD-3126](https://2u-internal.atlassian.net/browse/PROD-3126)

## Description

This PR adds a management command to upload course tags from csv

Note that we disable the test_list_query test since it is failing on CI. The test passes on our local environment. We are unable to find why the test fails on CI. We intend to debug it later.

## Testing

1. Add a BulkUploadCourseTagsConfig object in Admin using the csv. Also mark the enabled flag.
2. Run `./manage.py bulk_upload_tags`
3. Verify in admin or publisher that the tags have been updated

You can also pass the path of csv as argument to the command like `./manage.py bulk_upload_tags --csv_path=test.csv`
